pkgs/desktops/arcan/wrapper.nix

```diff
@@ -0,0 +1,28 @@
+{ arcan
+, makeWrapper
+, symlinkJoin
+, appls ? [ ]
+, name ? "arcan-wrapped"
+}:
+
+symlinkJoin rec {
+  inherit name;
+
+  paths = appls ++ [ arcan ];
```

Comment:
appls?

Reply:
Arcan lingo for "applications". It can go from a whole window manager to a simple terminal emulator.